app/components/work_packages/share/modal_body_component.rb

```diff
@@ -45,20 +47,39 @@ def self.wrapper_key
 
       private
 
-      def shared_users
-        @shared_users ||= User
-                            .having_entity_membership(@work_package)
-                            .includes(work_package_shares: :roles)
-                            .where(work_package_shares: { entity: @work_package })
-                            .order('work_package_shares.created_at DESC')
+      attr_reader :shared_users
+
+      def insert_target_modified?
+        true
+      end
+
+      def insert_target_modifier_id
+        'op-share-wp-active-shares'
+      end
+
+      # There is currently no available system argument for setting an id on the
+      # rendered <ul> tag that houses the row slots on Primer::Beta::BorderBox components.
+      def invited_user_list(&)
+        border_box = Primer::Beta::BorderBox.new
+
+        set_id_on_list_element(border_box)
+
+        render(border_box, &)
       end
 
-      def sharing_manageable?
-        User.current.allowed_to?(:share_work_packages, @work_package.project)
+      def set_id_on_list_element(list_container)
+        new_list_arguments = list_container.instance_variable_get(:@list_arguments)
+                                           .merge(id: insert_target_modifier_id)
+
+        list_container.instance_variable_set(:@list_arguments, new_list_arguments)
       end
 
-      def share_editable?(share)
-        User.current != share.principal && sharing_manageable?
+      def find_shared_users
+        @shared_users = User
+                          .having_entity_membership(@work_package)
+                          .includes(work_package_shares: :roles)
+                          .where(work_package_shares: { entity: @work_package })
+                          .ordered_by_name
```

Comment:
This is a bit weird to me. The method is [called once like this](https://github.com/opf/openproject/pull/13845/files#diff-dcc2c760c3da0ae92ef0691162adad19cc81662a330b2a062405da3c68fa139fR41)

```
@shared_users = find_shared_users
```

I'd thus remove the variable assignment here.